Vercel-sandbox-guide.html

```
@@ -0,0 +1,2303 @@
+
+        <div class="steps">
+          <!-- Step 1 -->
+          <div class="step">
+            <div class="step-num">1</div>
+            <div class="step-body">
+              <div class="step-title">Vercel CLI のインストール</div>
+              <div class="step-desc">Vercel CLI をグローバルインストールします。</div>
+              <div class="code-wrap">
+                <div class="code-header">
+                  <span class="code-lang">bash</span
+                  ><button class="copy-btn" onclick="copyCode(this)">Copy</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)